src/main/kotlin/racingcar/domain/race/CarRace.kt

```diff
@@ -0,0 +1,43 @@
+package racingcar.domain
+
+import racingcar.domain.car.Car
+import racingcar.domain.car.Cars
+import racingcar.domain.moving.RandomBasedMoveStrategy
+import racingcar.domain.record.RaceRecord
+
+class CarRace {
+    private val raceRecord = RaceRecord()
+
+    fun start(carCount: CarCount, trialCount: TrialCount): RaceRecord =
+        race(
+            initCars(carCount), trialCount
+        )
+
+    private fun initCars(carCount: CarCount): Cars =
+        Cars(
+            (0 until carCount.value).map {
+                Car(RandomBasedMoveStrategy())
+            }
+        )
```

Comment:
RandomBasedMoveStrategy는 매 Car를 만들 때마다 생성될 필요는 없어보입니다.
하나만 만들어두고 재사용하는 것은 어떨까요?

Reply:
또, 코틀린에서 제공해주는 함수들을 활용하면 아래처럼 작성해볼 수 있습니다 :)
```suggestion
    private fun initCars(carCount: CarCount): Cars =
        Cars(List(carCount.value) {Car(...) })
```